ygen/protogen.go

```diff
@@ -81,21 +81,27 @@ type protoMsg struct {
 	YANGPath string                   // YANGPath stores the path that the message corresponds to within the YANG schema.
 	Fields   []*protoMsgField         // Fields is a slice of the fields that are within the message.
 	Imports  []string                 // Imports is a slice of strings that contains the relative import paths that are required by this message.
-	Enums    map[string]*protoMsgEnum // Embedded enumerations within the message.
+	Enums    map[string]*protoMsgEnum // Enums lists the embedded enumerations within the message.
 }
 
 // protoMsgEnum represents an embedded enumeration within a protobuf message.
 type protoMsgEnum struct {
-	Values map[int64]string // The values that the enumerated type can take.
+	Values map[int64]protoEnumValue // The values that the enumerated type can take.
```

Comment:
Values that the enumerated... to avoid lint issues.

Reply:
Done.